ops/lib/__init__.py

```diff
@@ -108,53 +113,92 @@ def _find_all_specs(path):
             try:
                 lib_dirs = os.listdir(opslib)
             except OSError:
+                # logger.trace(...) # *lots* of things checked here
                 continue
+            else:
+                logger.debug("Trying %r.", opslib)
```

Comment:
the problem with logging in the exception is that you get a _lot_ of errors, most of them silly and spurious because it's easier to just try to get the directory. For example, my next comment will list a full run with the default PYTHONPATH on my system.

Reply:
```
    DEBUG    Looking for ops.lib packages under '.'.
    DEBUG    Tried './ops/opslib' but got [Errno 2] No such file or directory: './ops/opslib'
    DEBUG    Tried './docs/opslib' but got [Errno 2] No such file or directory: './docs/opslib'
    DEBUG    Tried './.flake8/opslib' but got [Errno 20] Not a directory: './.flake8/opslib'
    DEBUG    Tried './CODE_OF_CONDUCT.md/opslib' but got [Errno 20] Not a directory: './CODE_OF_CONDUCT.md/opslib'
    DEBUG    Tried './.travis.yml/opslib' but got [Errno 20] Not a directory: './.travis.yml/opslib'
    DEBUG    Tried './README.md/opslib' but got [Errno 20] Not a directory: './README.md/opslib'
    DEBUG    Tried './requirements-dev.txt/opslib' but got [Errno 20] Not a directory: './requirements-dev.txt/opslib'
    DEBUG    Tried './.mypy_cache/opslib' but got [Errno 2] No such file or directory: './.mypy_cache/opslib'
    DEBUG    Tried './.readthedocs.yaml/opslib' but got [Errno 20] Not a directory: './.readthedocs.yaml/opslib'
    DEBUG    Tried './.pytest_cache/opslib' but got [Errno 2] No such file or directory: './.pytest_cache/opslib'
    DEBUG    Tried './run_tests/opslib' but got [Errno 20] Not a directory: './run_tests/opslib'
    DEBUG    Tried './build_docs/opslib' but got [Errno 20] Not a directory: './build_docs/opslib'
    DEBUG    Tried './requirements.txt/opslib' but got [Errno 20] Not a directory: './requirements.txt/opslib'
    DEBUG    Tried './.git/opslib' but got [Errno 2] No such file or directory: './.git/opslib'
    DEBUG    Tried './test/opslib' but got [Errno 2] No such file or directory: './test/opslib'
    DEBUG    Tried './LICENSE.txt/opslib' but got [Errno 20] Not a directory: './LICENSE.txt/opslib'
    DEBUG    Tried './setup.py/opslib' but got [Errno 20] Not a directory: './setup.py/opslib'
    DEBUG    Tried './.gitignore/opslib' but got [Errno 20] Not a directory: './.gitignore/opslib'
    DEBUG    Tried './meh/opslib' but got [Errno 2] No such file or directory: './meh/opslib'
    DEBUG    Looking for ops.lib packages under '…'.
    DEBUG    Trying '…/some/opslib'.
    DEBUG    Found 'some.opslib.devel'.
    DEBUG    Parsing 'some.opslib.devel':
    DEBUG      Reached end without finding API, AUTHOR, PATCH, NAME.
    DEBUG    Found 'some.opslib.stable'.
    DEBUG    Parsing 'some.opslib.stable':
    DEBUG      Failed (PermissionError): [Errno 13] Permission denied: '…/some/opslib/stable/__init__.py'.
    DEBUG    Tried '…/README.md/opslib' but got [Errno 20] Not a directory: '…/README.md/opslib'
    DEBUG    Tried '…/requirements.txt/opslib' but got [Errno 20] Not a directory: '…/requirements.txt/opslib'
    DEBUG    Tried '…/tests/opslib' but got [Errno 2] No such file or directory: '…/tests/opslib'
    DEBUG    Tried '…/setup.py/opslib' but got [Errno 20] Not a directory: '…/setup.py/opslib'
    DEBUG    Looking for ops.lib packages under '/usr/lib/python3.8'.
    DEBUG    Tried '/usr/lib/python3.8/pickle.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pickle.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/unittest/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/unittest/opslib'
    DEBUG    Tried '/usr/lib/python3.8/plistlib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/plistlib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/fractions.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/fractions.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/codecs.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/codecs.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/ast.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/ast.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pdb.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pdb.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/nturl2path.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/nturl2path.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_dummy_thread.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_dummy_thread.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/configparser.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/configparser.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/mailcap.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/mailcap.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/ntpath.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/ntpath.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/dbm/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/dbm/opslib'
    DEBUG    Tried '/usr/lib/python3.8/zipimport.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/zipimport.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/mimetypes.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/mimetypes.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/gettext.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/gettext.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/shlex.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/shlex.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/quopri.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/quopri.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/hashlib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/hashlib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/wsgiref/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/wsgiref/opslib'
    DEBUG    Tried '/usr/lib/python3.8/telnetlib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/telnetlib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/tracemalloc.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/tracemalloc.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_threading_local.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_threading_local.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/distutils/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/distutils/opslib'
    DEBUG    Tried '/usr/lib/python3.8/getopt.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/getopt.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lzma.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lzma.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/copyreg.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/copyreg.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/http/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/http/opslib'
    DEBUG    Tried '/usr/lib/python3.8/multiprocessing/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/multiprocessing/opslib'
    DEBUG    Tried '/usr/lib/python3.8/ctypes/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/ctypes/opslib'
    DEBUG    Tried '/usr/lib/python3.8/code.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/code.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/selectors.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/selectors.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/venv/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/venv/opslib'
    DEBUG    Tried '/usr/lib/python3.8/enum.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/enum.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sndhdr.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/sndhdr.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sunau.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/sunau.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/traceback.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/traceback.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pydoc.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pydoc.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pstats.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pstats.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/functools.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/functools.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/cgitb.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/cgitb.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/asyncore.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/asyncore.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/colorsys.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/colorsys.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sched.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/sched.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/opcode.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/opcode.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/rlcompleter.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/rlcompleter.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/typing.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/typing.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/stat.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/stat.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/wave.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/wave.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/logging/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/logging/opslib'
    DEBUG    Tried '/usr/lib/python3.8/modulefinder.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/modulefinder.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/tarfile.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/tarfile.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/reprlib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/reprlib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/binhex.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/binhex.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/string.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/string.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/xml/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/xml/opslib'
    DEBUG    Tried '/usr/lib/python3.8/signal.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/signal.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/types.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/types.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_strptime.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_strptime.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/trace.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/trace.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/statistics.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/statistics.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_bootlocale.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_bootlocale.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/operator.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/operator.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/asynchat.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/asynchat.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/platform.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/platform.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/html/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/html/opslib'
    DEBUG    Tried '/usr/lib/python3.8/ftplib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/ftplib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/runpy.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/runpy.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/tkinter/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/tkinter/opslib'
    DEBUG    Tried '/usr/lib/python3.8/dis.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/dis.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/uuid.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/uuid.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/doctest.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/doctest.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/contextvars.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/contextvars.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/keyword.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/keyword.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/bisect.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/bisect.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/compileall.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/compileall.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_pyio.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_pyio.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/contextlib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/contextlib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_sysconfigdata__x86_64-linux-gnu.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_sysconfigdata__x86_64-linux-gnu.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/threading.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/threading.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_py_abc.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_py_abc.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/locale.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/locale.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/bdb.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/bdb.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/tokenize.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/tokenize.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pkgutil.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pkgutil.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/shelve.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/shelve.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/antigravity.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/antigravity.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/smtpd.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/smtpd.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/genericpath.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/genericpath.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/difflib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/difflib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/xmlrpc/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/xmlrpc/opslib'
    DEBUG    Tried '/usr/lib/python3.8/csv.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/csv.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_sitebuiltins.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_sitebuiltins.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/datetime.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/datetime.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/calendar.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/calendar.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/profile.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/profile.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/fnmatch.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/fnmatch.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pprint.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pprint.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_compat_pickle.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_compat_pickle.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/glob.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/glob.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/fileinput.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/fileinput.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/tempfile.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/tempfile.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/concurrent/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/concurrent/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_sysconfigdata_d_x86_64-linux-gnu.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_sysconfigdata_d_x86_64-linux-gnu.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sysconfig.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/sysconfig.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/optparse.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/optparse.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/formatter.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/formatter.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/py_compile.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/py_compile.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/codeop.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/codeop.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/cmd.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/cmd.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/gzip.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/gzip.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/site.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/site.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/symtable.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/symtable.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/posixpath.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/posixpath.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/dummy_threading.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/dummy_threading.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/zipfile.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/zipfile.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/stringprep.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/stringprep.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/config-3.8d-x86_64-linux-gnu/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/config-3.8d-x86_64-linux-gnu/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_pydecimal.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_pydecimal.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/secrets.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/secrets.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/ipaddress.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/ipaddress.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/smtplib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/smtplib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/zipapp.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/zipapp.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/timeit.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/timeit.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/netrc.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/netrc.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/subprocess.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/subprocess.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_collections_abc.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_collections_abc.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/socket.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/socket.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/io.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/io.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/copy.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/copy.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/random.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/random.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/weakref.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/weakref.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/re.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/re.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/turtle.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/turtle.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pipes.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pipes.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/tty.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/tty.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/dataclasses.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/dataclasses.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/textwrap.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/textwrap.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/shutil.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/shutil.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/mailbox.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/mailbox.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pyclbr.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pyclbr.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/webbrowser.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/webbrowser.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/urllib/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/urllib/opslib'
    DEBUG    Tried '/usr/lib/python3.8/socketserver.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/socketserver.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/bz2.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/bz2.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/uu.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/uu.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pathlib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pathlib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/linecache.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/linecache.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_weakrefset.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_weakrefset.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/imp.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/imp.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sqlite3/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/sqlite3/opslib'
    DEBUG    Tried '/usr/lib/python3.8/json/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/json/opslib'
    DEBUG    Tried '/usr/lib/python3.8/nntplib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/nntplib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sre_parse.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/sre_parse.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib2to3/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/lib2to3/opslib'
    DEBUG    Tried '/usr/lib/python3.8/test/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/test/opslib'
    DEBUG    Tried '/usr/lib/python3.8/LICENSE.txt/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/LICENSE.txt/opslib'
    DEBUG    Tried '/usr/lib/python3.8/heapq.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/heapq.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/numbers.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/numbers.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pickletools.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pickletools.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/xdrlib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/xdrlib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/crypt.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/crypt.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/config-3.8-x86_64-linux-gnu/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/config-3.8-x86_64-linux-gnu/opslib'
    DEBUG    Tried '/usr/lib/python3.8/base64.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/base64.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/collections/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/collections/opslib'
    DEBUG    Tried '/usr/lib/python3.8/email/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/email/opslib'
    DEBUG    Tried '/usr/lib/python3.8/abc.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/abc.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/symbol.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/symbol.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_osx_support.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_osx_support.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/tabnanny.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/tabnanny.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/importlib/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/importlib/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_markupbase.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_markupbase.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/os.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/os.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/getpass.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/getpass.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/lib-dynload/opslib'
    DEBUG    Tried '/usr/lib/python3.8/struct.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/struct.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/decimal.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/decimal.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sre_compile.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/sre_compile.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/cProfile.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/cProfile.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sre_constants.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/sre_constants.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_sysconfigdata__linux_x86_64-linux-gnu.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_sysconfigdata__linux_x86_64-linux-gnu.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/aifc.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/aifc.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/asyncio/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/asyncio/opslib'
    DEBUG    Tried '/usr/lib/python3.8/site-packages/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/site-packages/opslib'
    DEBUG    Tried '/usr/lib/python3.8/_compression.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/_compression.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pty.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/pty.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/imghdr.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/imghdr.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/warnings.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/warnings.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/poplib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/poplib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/argparse.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/argparse.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/inspect.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/inspect.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/__future__.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/__future__.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/chunk.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/chunk.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/this.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/this.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/encodings/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/encodings/opslib'
    DEBUG    Tried '/usr/lib/python3.8/queue.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/queue.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/curses/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/curses/opslib'
    DEBUG    Tried '/usr/lib/python3.8/token.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/token.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/hmac.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/hmac.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/ssl.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/ssl.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/cgi.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/cgi.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/filecmp.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/filecmp.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/__phello__.foo.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/__phello__.foo.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/imaplib.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/imaplib.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/sitecustomize.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/sitecustomize.py/opslib'
    DEBUG    Tried '/usr/lib/python3.8/pydoc_data/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/pydoc_data/opslib'
    DEBUG    Tried '/usr/lib/python3.8/ensurepip/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/ensurepip/opslib'
    DEBUG    Tried '/usr/lib/python3.8/__pycache__/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3.8/__pycache__/opslib'
    DEBUG    Looking for ops.lib packages under '/usr/lib/python3.8/lib-dynload'.
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testmultiphase.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testmultiphase.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_json.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_json.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_uuid.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_uuid.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testmultiphase.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testmultiphase.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_decimal.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_decimal.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_kr.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_kr.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_json.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_json.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_contextvars.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_contextvars.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/mmap.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/mmap.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_decimal.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_decimal.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_queue.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_queue.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_multibytecodec.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_multibytecodec.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_jp.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_jp.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/readline.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/readline.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_ctypes.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_ctypes.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_ctypes_test.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_ctypes_test.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/xxlimited.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/xxlimited.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_hashlib.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_hashlib.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_lzma.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_lzma.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_kr.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_kr.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/nis.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/nis.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testbuffer.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testbuffer.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_opcode.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_opcode.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_crypt.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_crypt.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_xxsubinterpreters.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_xxsubinterpreters.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/ossaudiodev.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/ossaudiodev.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_tkinter.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_tkinter.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_posixshmem.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_posixshmem.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_gdbm.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_gdbm.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_tw.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_tw.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_dbm.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_dbm.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_posixshmem.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_posixshmem.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_sqlite3.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_sqlite3.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_opcode.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_opcode.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testimportmultiple.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testimportmultiple.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testinternalcapi.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testinternalcapi.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testbuffer.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testbuffer.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_asyncio.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_asyncio.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testcapi.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testcapi.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/nis.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/nis.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_sqlite3.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_sqlite3.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/mmap.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/mmap.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_hk.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_hk.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_xxtestfuzz.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_xxtestfuzz.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_multiprocessing.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_multiprocessing.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_lzma.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_lzma.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_curses.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_curses.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/termios.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/termios.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/resource.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/resource.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_bz2.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_bz2.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_cn.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_cn.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/ossaudiodev.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/ossaudiodev.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_hk.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_hk.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/parser.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/parser.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_lsprof.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_lsprof.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_xxtestfuzz.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_xxtestfuzz.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_curses_panel.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_curses_panel.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_jp.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_jp.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_iso2022.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_iso2022.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_curses.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_curses.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_multibytecodec.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_multibytecodec.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/termios.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/termios.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_queue.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_queue.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_curses_panel.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_curses_panel.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_ctypes_test.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_ctypes_test.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/audioop.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/audioop.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_lsprof.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_lsprof.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_asyncio.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_asyncio.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_ssl.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_ssl.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/audioop.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/audioop.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/resource.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/resource.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_hashlib.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_hashlib.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_ctypes.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_ctypes.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_bz2.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_bz2.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_contextvars.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_contextvars.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_cn.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_cn.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testimportmultiple.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testimportmultiple.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_multiprocessing.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_multiprocessing.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testinternalcapi.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testinternalcapi.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_tw.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_tw.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_testcapi.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_testcapi.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_uuid.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_uuid.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/readline.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/readline.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_xxsubinterpreters.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_xxsubinterpreters.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_codecs_iso2022.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_codecs_iso2022.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/parser.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/parser.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_ssl.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_ssl.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_dbm.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_dbm.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3.8/lib-dynload/_crypt.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3.8/lib-dynload/_crypt.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Looking for ops.lib packages under '/usr/local/lib/python3.8/dist-packages'.
    DEBUG    Looking for ops.lib packages under '/usr/lib/python3/dist-packages'.
    DEBUG    Tried '/usr/lib/python3/dist-packages/drv_libxml2.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/drv_libxml2.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libpasteurize/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/libpasteurize/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/indicator_cpufreq/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/indicator_cpufreq/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/zipp-1.0.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/zipp-1.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/HweSupportStatus/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/HweSupportStatus/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/checkbox_support/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/checkbox_support/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/systemd/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/systemd/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Quirks/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Quirks/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/readme_renderer/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/readme_renderer/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/PIL/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/PIL/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/docs/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/docs/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/_version.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/_version.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/devscripts-2.20.2ubuntu2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/devscripts-2.20.2ubuntu2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/python_apt-2.0.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/python_apt-2.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/speechd_config/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/speechd_config/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/uno.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/uno.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/distlib/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/distlib/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/macaroonbakery-1.3.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/macaroonbakery-1.3.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/future-0.18.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/future-0.18.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/debconf.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/debconf.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/distro_info-0.23ubuntu1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/distro_info-0.23ubuntu1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/python_termstyle-0.1.10.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/python_termstyle-0.1.10.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pycodestyle-2.5.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pycodestyle-2.5.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/filelock-3.0.12.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/filelock-3.0.12.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pytz-2019.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pytz-2019.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/python_gitlab-2.0.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/python_gitlab-2.0.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/bleach/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/bleach/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/virtualenv-20.0.17.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/virtualenv-20.0.17.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/requests_unixsocket-0.2.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/requests_unixsocket-0.2.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/importlib_metadata-1.5.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/importlib_metadata-1.5.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/LanguageSelector/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/LanguageSelector/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/language_support_pkgs.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/language_support_pkgs.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/six-1.14.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/six-1.14.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/autopep8-1.5.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/autopep8-1.5.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/psutil-5.5.1.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/psutil-5.5.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/dulwich/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/dulwich/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apport/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/apport/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/python_dateutil-2.7.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/python_dateutil-2.7.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/importlib_metadata/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/importlib_metadata/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apt_pkg-stubs/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/apt_pkg-stubs/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/distro.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/distro.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/xdiagnose/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/xdiagnose/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Pillow-7.0.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Pillow-7.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/guacamole-0.9.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/guacamole-0.9.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ufw/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ufw/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/onboard-1.4.1.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/onboard-1.4.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/quodlibet-4.2.1.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/quodlibet-4.2.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyflakes/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyflakes/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/easy_install.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/easy_install.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/debian_bundle/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/debian_bundle/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cryptography/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/cryptography/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/keyring/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/keyring/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/CommandNotFound/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/CommandNotFound/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/mutagen/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/mutagen/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/html5lib-1.0.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/html5lib-1.0.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/_cffi_backend.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/_cffi_backend.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pip-20.0.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pip-20.0.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/jeepney/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/jeepney/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/nose-1.3.7.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/nose-1.3.7.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cups.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/cups.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/oauthlib/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/oauthlib/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/httplib2-0.14.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/httplib2-0.14.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/deprecated/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/deprecated/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/python_debian-0.1.36ubuntu1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/python_debian-0.1.36ubuntu1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/softwareproperties/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/softwareproperties/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/breezy-3.0.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/breezy-3.0.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/lazr.uri-1.0.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/lazr.uri-1.0.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/readme_renderer-24.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/readme_renderer-24.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/prompt_toolkit/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/prompt_toolkit/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/tabulate.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/tabulate.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pkginfo-1.4.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pkginfo-1.4.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/validate.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/validate.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/caffeine-2.9.4.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/caffeine-2.9.4.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/launchpadlib-1.10.13.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/launchpadlib-1.10.13.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libvirtmod_lxc.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libvirtmod_lxc.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/clonevirtualenv.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/clonevirtualenv.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Markdown-3.1.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Markdown-3.1.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyflakes-2.1.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyflakes-2.1.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/rednose.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/rednose.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pkg_resources/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pkg_resources/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/olefile-0.46.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/olefile-0.46.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyasn1-0.4.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyasn1-0.4.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/systemd_python-234.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/systemd_python-234.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/gpg/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/gpg/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/requests/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/requests/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/traitlets/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/traitlets/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pip/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pip/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/geisview/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/geisview/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/feedparser.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/feedparser.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/SQLAlchemy-1.3.12.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/SQLAlchemy-1.3.12.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/soupsieve/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/soupsieve/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ssh_import_id-5.10.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ssh_import_id-5.10.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/hpmudext.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/hpmudext.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/command_not_found-0.3.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/command_not_found-0.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cryptography-2.8.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/cryptography-2.8.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/markupsafe/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/markupsafe/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/secretstorage/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/secretstorage/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cupshelpers-1.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/cupshelpers-1.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/dulwich-0.19.15.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/dulwich-0.19.15.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pycups-1.9.73.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pycups-1.9.73.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/_dbus_glib_bindings.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/_dbus_glib_bindings.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/aptsources/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/aptsources/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/magic/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/magic/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/soupsieve-1.9.5.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/soupsieve-1.9.5.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/idna-2.8.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/idna-2.8.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ndiff.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/ndiff.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/parso/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/parso/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/tqdm/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/tqdm/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/googleapiclient/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/googleapiclient/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/keyrings.alt-3.4.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/keyrings.alt-3.4.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/wrapt/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/wrapt/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/speechd/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/speechd/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/packaging/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/packaging/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/markdown/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/markdown/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/chardet-3.0.4.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/chardet-3.0.4.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pipenv/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pipenv/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/aptdaemon/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/aptdaemon/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/screen_resolution_extra-0.0.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/screen_resolution_extra-0.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/bs4/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/bs4/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/requests_toolbelt/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/requests_toolbelt/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/twine-3.1.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/twine-3.1.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pycurl-7.43.0.2.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pycurl-7.43.0.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Crypto/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Crypto/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/_dbus_bindings.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/_dbus_bindings.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/google_auth-1.5.1-nspkg.pth/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/google_auth-1.5.1-nspkg.pth/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/decorator.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/decorator.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/crcmod-1.7.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/crcmod-1.7.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/past/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/past/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/requests_unixsocket/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/requests_unixsocket/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ipython_genutils-0.2.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/ipython_genutils-0.2.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/future/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/future/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cupsext.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/cupsext.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/monotonic.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/monotonic.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/dbus_python-1.2.16.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/dbus_python-1.2.16.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/devscripts/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/devscripts/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/appdirs.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/appdirs.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyRFC3339-1.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyRFC3339-1.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Brlapi-0.7.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/Brlapi-0.7.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/jeepney-0.4.2.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/jeepney-0.4.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/PyGObject-3.36.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/PyGObject-3.36.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/unohelper.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/unohelper.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Jinja2-2.10.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Jinja2-2.10.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Deprecated-1.2.7.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Deprecated-1.2.7.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pycrypto-2.6.1.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pycrypto-2.6.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/deb822.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/deb822.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libvirt.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libvirt.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/bleach-3.1.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/bleach-3.1.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/mpmath/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/mpmath/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/fasteners/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/fasteners/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/unidiff-0.5.5.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/unidiff-0.5.5.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/paramiko-2.6.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/paramiko-2.6.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/xlsxwriter/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/xlsxwriter/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/unidiff/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/unidiff/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/yaml/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/yaml/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Pygments-2.3.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Pygments-2.3.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/wheel/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/wheel/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/curl/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/curl/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/DistUpgrade/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/DistUpgrade/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyxdg-0.26.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pyxdg-0.26.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/usbcreator/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/usbcreator/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/lockfile-0.12.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/lockfile-0.12.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/tabulate-0.8.6.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/tabulate-0.8.6.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/docutils/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/docutils/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apt/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/apt/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/scanext.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/scanext.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/distro_info.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/distro_info.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/urllib3-1.25.8.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/urllib3-1.25.8.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/jedi/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/jedi/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/olefile/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/olefile/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/guacamole/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/guacamole/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/traitlets-4.3.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/traitlets-4.3.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pipenv-11.9.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pipenv-11.9.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/httplib2/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/httplib2/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyrfc3339/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyrfc3339/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/certifi-2019.11.28.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/certifi-2019.11.28.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/unattended_upgrades-0.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/unattended_upgrades-0.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/entrypoints.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/entrypoints.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/uaclient/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/uaclient/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/crcmod/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/crcmod/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/rsa/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/rsa/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/oauthlib-3.1.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/oauthlib-3.1.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/sympy-1.5.1.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/sympy-1.5.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/yapftests/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/yapftests/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/OpenSSL/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/OpenSSL/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyatspi/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyatspi/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/nose/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/nose/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/xkit-0.0.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/xkit-0.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cachetools/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/cachetools/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/PyYAML-5.3.1.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/PyYAML-5.3.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/webencodings/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/webencodings/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/lockfile/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/lockfile/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apturl-0.5.2.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/apturl-0.5.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pycodestyle.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pycodestyle.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libvirt_lxc.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libvirt_lxc.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/roman-2.0.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/roman-2.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/PyNaCl-1.3.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/PyNaCl-1.3.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/language_selector-0.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/language_selector-0.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/gitlab/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/gitlab/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/jinja2/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/jinja2/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/keyrings/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/keyrings/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libxml2.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libxml2.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/filelock.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/filelock.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libxml2mod.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libxml2mod.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/wrapt-1.11.2.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/wrapt-1.11.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/decorator-4.4.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/decorator-4.4.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/fastimport/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/fastimport/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/feedparser-5.2.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/feedparser-5.2.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/duplicity/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/duplicity/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/wheel-0.34.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/wheel-0.34.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/python_distutils_extra-2.39.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/python_distutils_extra-2.39.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/geis/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/geis/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/jedi-0.15.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/jedi-0.15.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/iotop-0.6.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/iotop-0.6.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/quodlibet/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/quodlibet/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libvirtmod_qemu.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libvirtmod_qemu.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/lsb_release.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/lsb_release.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/urllib3/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/urllib3/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/more_itertools-4.2.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/more_itertools-4.2.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/requests_toolbelt-0.8.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/requests_toolbelt-0.8.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/macaroonbakery/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/macaroonbakery/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/lxml-4.5.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/lxml-4.5.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyOpenSSL-19.0.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyOpenSSL-19.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/bcrypt/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/bcrypt/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libsmbios_c/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/libsmbios_c/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/idna/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/idna/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/requests-2.22.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/requests-2.22.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/webencodings-0.5.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/webencodings-0.5.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/_geis_bindings.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/_geis_bindings.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/UbuntuDrivers/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/UbuntuDrivers/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/mako/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/mako/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/isympy.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/isympy.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyasn1/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyasn1/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ubuntu_drivers_common-0.0.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ubuntu_drivers_common-0.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/netifaces.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/netifaces.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/lazr.restfulclient-0.14.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/lazr.restfulclient-0.14.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/docutils-0.16.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/docutils-0.16.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Xlib/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Xlib/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pygtkcompat/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pygtkcompat/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/lxml/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/lxml/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/defer-1.0.6.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/defer-1.0.6.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/DistUtilsExtra/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/DistUtilsExtra/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/roman.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/roman.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/google_auth-1.5.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/google_auth-1.5.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/sympy/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/sympy/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apt_pkg.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/apt_pkg.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/rsa-4.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/rsa-4.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/setuptools/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/setuptools/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/louis/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/louis/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/usb_creator-0.3.7.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/usb_creator-0.3.7.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pymacaroons-0.13.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pymacaroons-0.13.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apt_inst-stubs/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/apt_inst-stubs/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/checkbox_support-0.22.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/checkbox_support-0.22.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/prompt_toolkit-2.0.10.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/prompt_toolkit-2.0.10.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/progressbar-2.5.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/progressbar-2.5.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/beautifulsoup4-4.8.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/beautifulsoup4-4.8.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/scour-0.37.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/scour-0.37.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/termstyle.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/termstyle.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apt_inst.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/apt_inst.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libfuturize/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/libfuturize/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/yapf/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/yapf/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pickleshare.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pickleshare.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pickleshare-0.7.5.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pickleshare-0.7.5.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/html5lib/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/html5lib/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/github/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/github/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/padme/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/padme/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/google_api_python_client-1.7.11.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/google_api_python_client-1.7.11.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/talloc.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/talloc.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/google/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/google/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libvirt_python-6.1.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libvirt_python-6.1.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/coverage/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/coverage/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/fasteners-0.14.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/fasteners-0.14.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/oauth2client-4.1.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/oauth2client-4.1.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/coverage-4.5.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/coverage-4.5.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/plainbox/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/plainbox/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/_yaml.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/_yaml.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/musicbrainzngs-0.7.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/musicbrainzngs-0.7.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/chardet/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/chardet/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Onboard/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Onboard/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/packaging-20.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/packaging-20.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/entrypoints.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/entrypoints.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/dell_recovery-0.0.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/dell_recovery-0.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/gi/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/gi/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apport_python_hook.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/apport_python_hook.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/dateutil/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/dateutil/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ufw-0.36.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/ufw-0.36.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/more_itertools/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/more_itertools/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/dbus/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/dbus/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/configobj-5.0.6.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/configobj-5.0.6.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/psutil/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/psutil/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyparsing.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pyparsing.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/problem_report.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/problem_report.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/SecretStorage-3.1.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/SecretStorage-3.1.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/launchpadlib/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/launchpadlib/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cupshelpers/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/cupshelpers/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/breezy/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/breezy/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/simplejson-3.16.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/simplejson-3.16.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/backcall-0.1.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/backcall-0.1.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libvirtaio.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libvirtaio.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/iotop/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/iotop/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Mako-1.1.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Mako-1.1.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pexpect/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pexpect/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/jwt/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/jwt/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/indicator_cpufreq-0.2.2.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/indicator_cpufreq-0.2.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/progressbar/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/progressbar/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/brlapi.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/brlapi.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyparsing-2.4.6.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyparsing-2.4.6.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/oauth2client/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/oauth2client/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/zipp.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/zipp.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/louis-3.12.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/louis-3.12.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pygments/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pygments/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pycairo-1.16.2.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pycairo-1.16.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/uritemplate-0.6.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/uritemplate-0.6.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/uritemplate/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/uritemplate/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/AptUrl/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/AptUrl/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ewmh/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ewmh/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/IPython/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/IPython/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/appdirs-1.4.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/appdirs-1.4.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/redshift_gtk/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/redshift_gtk/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pytz/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pytz/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/plainbox-0.25.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/plainbox-0.25.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/python_magic-0.4.16.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/python_magic-0.4.16.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/six.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/six.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/PyJWT-1.7.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/PyJWT-1.7.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/google_auth_httplib2.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/google_auth_httplib2.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/tqdm-4.30.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/tqdm-4.30.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/MarkupSafe-1.1.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/MarkupSafe-1.1.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pcardext.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pcardext.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/virtualenv/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/virtualenv/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/flake8/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/flake8/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/PyGithub-1.43.7.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/PyGithub-1.43.7.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/padme-1.1.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/padme-1.1.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/wadllib/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/wadllib/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/UpdateManager/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/UpdateManager/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyasn1_modules/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pyasn1_modules/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pexpect-4.6.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pexpect-4.6.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/reportlab/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/reportlab/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/mccabe.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/mccabe.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/wcwidth-0.1.8.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/wcwidth-0.1.8.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/setuptools-45.2.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/setuptools-45.2.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ipython_genutils/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ipython_genutils/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pyasn1_modules-0.2.1.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pyasn1_modules-0.2.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/flake8-3.7.9.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/flake8-3.7.9.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/netifaces-0.10.4.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/netifaces-0.10.4.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/xkit/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/xkit/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/NvidiaDetector/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/NvidiaDetector/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/keyring-18.0.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/keyring-18.0.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/mpmath-1.1.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/mpmath-1.1.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/reportlab-3.5.34.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/reportlab-3.5.34.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/netifaces.cpython-38d-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/netifaces.cpython-38d-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/protobuf-3.6.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/protobuf-3.6.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/defer/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/defer/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/duplicity-0.8.12.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/duplicity-0.8.12.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/bcrypt-3.1.7.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/bcrypt-3.1.7.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/wcwidth/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/wcwidth/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/mccabe-0.6.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/mccabe-0.6.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/twine/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/twine/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/fastimport-0.9.8.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/fastimport-0.9.8.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ptyprocess/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ptyprocess/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ipython-7.13.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ipython-7.13.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pkginfo/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pkginfo/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/certifi/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/certifi/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/nacl/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/nacl/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/monotonic-1.5.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/monotonic-1.5.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/chrome_gnome_shell-0.0.0.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/chrome_gnome_shell-0.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/rednose-0.4.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/rednose-0.4.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/system_service-0.3.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/system_service-0.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cairo/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/cairo/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/distlib-0.3.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/distlib-0.3.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/cachetools-4.0.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/cachetools-4.0.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/wadllib-1.3.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/wadllib-1.3.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/distro-1.4.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/distro-1.4.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/parso-0.5.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/parso-0.5.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/musicbrainzngs/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/musicbrainzngs/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libvirtmod.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libvirtmod.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/python_xlib-0.23.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/python_xlib-0.23.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/xdg/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/xdg/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/google_auth_httplib2-0.0.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/google_auth_httplib2-0.0.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pep8-1.7.1.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pep8-1.7.1.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ubuntu_advantage_tools-20.3.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ubuntu_advantage_tools-20.3.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/UbuntuSystemService/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/UbuntuSystemService/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/lazr/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/lazr/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/blinker/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/blinker/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ssh_import_id/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ssh_import_id/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/sqlalchemy/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/sqlalchemy/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/gtimelog/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/gtimelog/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/mutagen-1.44.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/mutagen-1.44.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/simplejson/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/simplejson/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/orca/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/orca/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/README.txt/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/README.txt/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/scour/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/scour/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/blinker-1.4.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/blinker-1.4.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/xdiagnose-3.8.9.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/xdiagnose-3.8.9.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/configobj.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/configobj.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/gtimelog-0.11.2.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/gtimelog-0.11.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pycurl.cpython-38-x86_64-linux-gnu.so/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pycurl.cpython-38-x86_64-linux-gnu.so/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/yapf-0.29.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/yapf-0.29.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/paramiko/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/paramiko/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/libvirt_qemu.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/libvirt_qemu.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pymacaroons/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/pymacaroons/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/ewmh-0.1.6.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/ewmh-0.1.6.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/autopep8.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/autopep8.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/janitor/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/janitor/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/XlsxWriter-1.1.2.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/XlsxWriter-1.1.2.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/Dell/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/Dell/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/feedparser_debian/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/feedparser_debian/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/gtweak/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/gtweak/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/apiclient/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/apiclient/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/backcall/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/backcall/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/pep8.py/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/pep8.py/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/virtualenv_clone-0.3.0.egg-info/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/virtualenv_clone-0.3.0.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/debian/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/debian/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/gpg-1.13.1_unknown.egg-info/opslib' but got [Errno 20] Not a directory: '/usr/lib/python3/dist-packages/gpg-1.13.1_unknown.egg-info/opslib'
    DEBUG    Tried '/usr/lib/python3/dist-packages/__pycache__/opslib' but got [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/__pycache__/opslib'

Reply:
I agree that is too many Errno 2 and Errno 20 items. Would it be worth filtering those out explicitly and logging the rest?

Reply:
d'oh. Of course.